doc/OnlineDocs/pyomo_contrib.rst

```diff
@@ -0,0 +1,74 @@
+Third-Party Contributions
+=========================
+
+Pyomo uses the ``pyomo.contrib`` package to facilitate the integrate
+the inclusion of third-party contributions that enhance Pyomo's
+core functionaliy.  The following two examples illustrate two ways
+that ``pyomo.contrib`` can be used to integrate third-party
+contributions.
+
+Including External Packages
+---------------------------
+
+The `pyomocontrib_simplemodel
```

Comment:
I would rather see this be documentation on how to implement a contrib package.  I think the actual documentation for simplemodel and example (even the brief overview) should be in their respective directories and linked from here.